api-2.2/src/main/java/org/openmrs/module/fhir/helper/ConditionHelperImpl2_2.java

```diff
@@ -68,6 +67,48 @@ public Condition createCondition(Condition condition) {
 		return generateFHIRCondition(openMrsCondition);
 	}
 
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#updateFHIRCondition(org.hl7.fhir.dstu3.model.Condition)
+	 */
+	@Override
+	public Condition updateFHIRCondition(Condition fhirCondition) {
+		if (fhirCondition.getId() != null) {
+			ConditionService conditionService = Context.getService(ConditionService.class);
+			org.openmrs.Condition openmrsCondition = conditionService.getConditionByUuid(fhirCondition.getId());
```

Comment:
When you say createCondition, does that mean some updates create new conditions instead of updating existing ones?

Reply:
Pick it from wiki documentation for fhir module. 
check this one for patient resource https://wiki.openmrs.org/display/projects/Patient+Resource
weird but it was implemented and documented

Reply:
Can we also document this for the condition resource?

Reply:
Sure I will
https://issues.openmrs.org/browse/FM-67